src/services/reviewsApi.ts

```diff
@@ -25,10 +26,21 @@ export const fetchReviews = async (sellerId:string) => {
 // Create a new review
 export const createReview = async (formData: FormData) => {
   try {
-    const response = await axiosClient.post('/review-feedback/add', formData, );
-    return response.data;
-  } catch (error) {
-    console.error('Error creating review: ', error);
+    const response = await axiosClient.post('/review-feedback/add', formData);
+    if (response.status === 200) {
+      return response.data;
+    }
+
+    if (response.status === 400) {
```

Comment:
I don’t think it’s necessary to send all these messages back to the user. Instead, we could simplify it to two messages: one to confirm that the review was successfully added and another to indicate if the review submission failed. We can log the detailed errors on our end.